### PR TITLE
feat(metadata): error add stack trace

### PR DESCRIPTION
**Pull request by lianxmfor**
relate #903

### CODE REVIEW

File: internal/database/metadata/mysql/entity.go [hunk 1]
```diff
@@ -14,15 +14,15 @@ func createEntity(ctx context.Context, sqlxCtx metadata.SqlxContext, opt metadat
 	if err != nil {
 		if er, ok := err.(*mysql.MySQLError); ok {
 			if er.Number == ER_DUP_ENTRY {
-				return 0, fmt.Errorf("entity %s already exists", opt.EntityName)
+				return 0, errors.Errorf("entity %s already exists", opt.EntityName)
 			}
 		}
 		return 0, err
```
Comment (wfxr) [inline internal/database/metadata/mysql/entity.go:20]: missing wrap ?

Comment (lianxmfor) [reply internal/database/metadata/mysql/entity.go:20]: `errors.Errorf` will automatically save the call stack, just like `errors.WithStack`.

Comment (lianxmfor) [reply internal/database/metadata/mysql/entity.go]: fixed

**Review by YikSanChan — APPROVED** [review]
lgtm

**Pull request merged**